modules/seqexec/model/src/main/scala/seqexec/model/ExecutionQueueView.scala

```diff
@@ -7,17 +7,24 @@ import cats.Eq
 import cats.implicits._
 import gem.Observation
 import monocle.macros.Lenses
-import seqexec.model.enum.{BatchCommandState, BatchExecState}
+import seqexec.model.enum.BatchCommandState
+import seqexec.model.enum.BatchExecState
 
 @Lenses
 final case class ExecutionQueueView(id:        QueueId,
                                     name:      String,
                                     cmdState:  BatchCommandState,
                                     execState: BatchExecState,
-                                    queue:     List[Observation.Id])
+                                    queue:     List[Observation.Id]) {
+  val observer: Option[Observer] = cmdState match {
```

Comment:
I'm not happy about this but it will do for now. Ideally we'd have the observer in the model for real